apps/sms/js/contacts.js

```diff
@@ -206,31 +200,27 @@
           results = contacts;
         }
 
-        callback(results, {
-          terms: terms
-        });
-      };
-
-      request.onerror = function onerror() {
-        // When an error occurs, regardless of completed count,
-        // clear the onsuccess handler from this request and immediately
-        // invoke the callback with a `null` argument.
-        this.onsuccess = this.onerror = null;
-        callback(null);
-      };
+        return results;
```

Comment:
I see you improved this code a bit (removed slice), so just on the side note - we can improve this callback further with for example:
```
if (terms.length > 1) {
  var fields = ['tel', 'givenName', 'familyName'];
  if (Settings.supportEmailRecipient) {
    fields.push('email');
  }
  var criteria = { fields: fields, terms: lower };
  return contacts.filter((contact) => isMatch(contact, criteria, filterFns.contains));
}

return contacts;
```

No need to do it in the patch - just if you want :)